spikeinterface/comparison/basecomparison.py

```diff
@@ -1,9 +1,38 @@
 import numpy as np
-import pandas as pd
-from .comparisontools import (do_count_event, make_match_count_matrix, make_agreement_scores_from_count)
+from .comparisontools import (make_possible_match, make_best_match, make_hungarian_match,
+                              do_count_event, make_match_count_matrix, make_agreement_scores_from_count)
 
 
 class BaseComparison:
+    """
+    Base class for all comparisons (SpikeTrain and Template)
+    """
+
+    def __init__(self, name_list,
+                 match_score=0.5, chance_score=0.1,
+                 verbose=False):
```

Comment:
I think that base comparison should have `sorting_list`

Reply:
No, because this could be also WaveformExtractors. We could have `object_list`